### PR TITLE
[14.0][MIG] account_cryptocurrency

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module account_cryptocurrency to v14

cc ~ @ForgeFlow

### CODE REVIEW

File: account_cryptocurrency/models/res_currency_move.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+# Copyright 2018 Fork Sand Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+_STATES = [
+    ("draft", "Draft"),
+    ("posted", "Posted"),
+    ("cancel", "Cancelled"),
```
Comment (MiquelRForgeFlow) [inline account_cryptocurrency/models/res_currency_move.py:11]: Selection_188

migration scripts are needed for this change

**Review by MiquelRForgeFlow — APPROVED** [review]

File: account_cryptocurrency/models/res_currency_move.py (hunk 1 above)
Comment (AdriaGForgeFlow) [reply account_cryptocurrency/models/res_currency_move.py:11]: Nice catch, added them :+1:

File: account_cryptocurrency/models/account_account.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        for rec in self.filtered("currency_id"):
+            currency = self.env["res.currency"].search(
+                [
+                    ("inventory_account_id", "=", rec.id),
+                    ("id", "!=", rec.currency_id.id),
+                ],
+                limit=1,
+            )
+            if currency:
+                raise exceptions.Warning(
```
Comment (MiquelRForgeFlow) [inline account_cryptocurrency/models/account_account.py]:
```suggestion
                raise exceptions.ValidationError(
```

**Review by MiquelRForgeFlow — COMMENTED** [review]
Apply the same in the other warning constrains

File: account_cryptocurrency/README.rst [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://odoo-community.org/logo.png>`_.
+
+Contributors
+------------
+
+* Jordi Ballester <jordi.ballester@eficent.com>
```
Comment (JordiBForgeFlow) [inline account_cryptocurrency/README.rst]: change to jordi.ballester@forgeflow.com
Add yourself as contributor.

File: account_cryptocurrency/readme/CONFIGURE.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+   Settings / Multi-Currencies.
+
+#. Create the cryptocurrency in 'Invoicing / Configuration / Accounting /
+   Currencies'. Complete the following fields:  'Inventoried', 'Valuation
+   Method'. Indicate the desired rounding factor.
+
+#. Add to the cryptocurrency an 'Inventory Account'. This Account must
+   indicate in field 'Account Currency' the same currency.
+
+#. Create an Account Journal associated to the payments of the crypto
```
Comment (JordiBForgeFlow) [inline account_cryptocurrency/readme/CONFIGURE.rst]: The account journal should be of type 'Bank'

File: account_cryptocurrency/readme/CONFIGURE.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+   currency, in 'Invoicing / Configuration / Accounting / Journals'. Indicate
+   in the 'Currency' field the cryptocurrency. In the 'Default Credit Account'
```
Comment (JordiBForgeFlow) [inline account_cryptocurrency/readme/CONFIGURE.rst]: There's no longer a default_debit and default_credit accounts. Should be expressed as 'Bank Account'. An Outstanding Receipts Account and Outstanding Payments Account is also required.

File: account_cryptocurrency/models/account_account.py (hunk 2 above)
Comment (AdriaGForgeFlow) [reply account_cryptocurrency/models/account_account.py]: Done, thanks!

**Review by JordiBForgeFlow — CHANGES_REQUESTED** [review]

File: account_cryptocurrency/README.rst [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+Configuration
+=============
```
Comment (MiquelRForgeFlow) [inline account_cryptocurrency/README.rst]:
```suggestion
```
In the readme/CONFIGURE.rst file, delete this

Comment (AdriaGForgeFlow) [reply account_cryptocurrency/README.rst]: Didn't notice, thanks!

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests :+1:

**Pull request merged**